Change calendar display dependent on mobile or large display

nkfoss opened the pull request:
Fixes #24

There was a nice fix found online that allows us to display certain elements, dependent on the screen size.

https://forum.squarespace.com/topic/29657-how-to-embed-a-google-calendar-in-a-responsive-way/

nkfoss committed:
Merge branch 'master' into Calendar

nkfoss committed:
Change calendar display dependent on mobile or large display

Fixes #24

There was a nice fix found online that allows us to display certain elements, dependent on the screen size.

Commit:
Merge branch 'master' into Calendar

michael-small reviewed (dismissed):
It is really narrow on desktop and goes offscreen on mobile. Looking into that now.

Commit:
Make calendar iframes responsively sized widths

We managed to make it responsive to month vs agenda view, but the styling had the calendars either go offscreen or use too little of the screen. Moving the iframes to `Content.js` and setting `iframe {width: 100%}` resolved the issue as far as I can tell.

It just occurred to me as I write this that this may make it harder to directly change the calendar in the CMS, but we could make a workaround if need be.
